frontend/public/lib/imageOperations.ts

```diff
@@ -83,13 +84,12 @@ export async function getCompressedJPG(file, maxSizeMB): Promise<string> {
       drawImageOnCanvas(image, canvas);
       canvas.toBlob(
         async function (blob) {
-          const options = {
-            maxSizeMB: maxSizeMB,
```

Comment:
Since maxSizeMB in not being used any longer it can be removed from the function parameters.